packages/firebase_ui_firestore/lib/src/query_builder.dart

```diff
@@ -451,7 +457,7 @@ class FirestoreListView<Document> extends FirestoreQueryBuilder<Document> {
   }) : super(
           key: key,
           query: query,
-          pageSize: pageSize,
+          pageSize: pageSize ?? (showLoadingIndicator ? 20 : 10),
```

Comment:
can you explain the logic behind this? I'm not sure why page size should be different depending on whether there's a loading indicator or not?

Reply:
I originally added this while testing, as height of my list items was really small and with the default 10, loading indicator was seldom showing. I'll restore the default page size.